test/run_test.py

```diff
@@ -516,6 +531,23 @@ def get_slow_tests_based_on_S3() -> List[str]:
     return slow_tests
 
 
+def get_test_case_args(test_module, using_pytest) -> List[str]:
+    if test_module not in SPECIFIED_TEST_CASES_DICT:
+        print('Warning! This should not happen as this should be checked before entering the function.')
```

Comment:
Could you modify this warning message so it says what "This" is? It seems like it would be confusing/uninformative if it ever does get printed